.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,63 @@
+name: Upload Python Package
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - name: Publish linux
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          docker run -v $PWD:/io quay.io/pypa/manylinux1_x86_64 "/io/build_wheels.sh"
+          pip install twine
```

Comment:
Probably better to move this one line up. Let cheap things fail first before expensive things.